test/datafmt.jl

```diff
@@ -93,6 +93,14 @@ let x = [1,2,3], y = [4,5,6], io = IOBuffer()
     @test readcsv(io) == [x y]
 end
 
+let x = [0.1 0.3 0.5], io = IOBuffer()
```

Comment:
Sorry, I can't find it.

Also, why not merge the two `let` blocks and call `writedlm` only once?